vulture/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.14'
```

Comment:
Please add `from core import Vulture` above __version__. As a side effect, we don't have to change the imports in the test files.